apps/web-giddh/src/app/services/group.service.ts

```
@@ -226,9 +231,26 @@ export class GroupService {
 
-    public getMasters(groupUniqueName: string, page: Number): Observable<BaseResponse<GroupResponse, string>> {
+    /**
+     * Gets master data for a group
+     * 
+     * @param {string} groupUniqueName Unique name of the group
+     * @param {Number} page Page number for pagination
+     * @param {string} [accountArchiveStatus] Filter for archived accounts
+     * @returns {Observable<BaseResponse<GroupResponse, string>>} Observable with group response data
+     * @memberof GroupService
+     */
+    public getMasters(groupUniqueName: string, page: Number, accountArchiveStatus?: string): Observable<BaseResponse<GroupResponse, string>> {
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)